sdk/messaging/azservicebus/internal/go-amqp/client.go

```diff
@@ -0,0 +1,203 @@
+// Copyright (C) 2017 Kale Blankenship
```

Comment:
@RickWinter  - an example of our "combined" copyright (obviously subject to whatever we need to change).